content/docs/support/cli/_index.md

```diff
@@ -845,23 +845,23 @@ dellctl images --component csi-vxflexos
 ```
 ```
 Driver/Module Image             Supported Orchestrator Versions         Sidecar Images
-dellemc/csi-vxflexos:v2.9.0     k8s1.28,k8s1.27,k8s1.26,ocp4.14,ocp4.13 registry.k8s.io/sig-storage/csi-attacher:v4.4.2
+quay.io/dell/container-storage-modules/csi-vxflexos:v2.12.0     k8s1.31,k8s1.30,k8s1.29,ocp4.17,ocp4.16 registry.k8s.io/sig-storage/csi-attacher:v4.4.2
                                                                         registry.k8s.io/sig-storage/csi-provisioner:v3.6.2
                                                                         registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.10.0
                                                                         registry.k8s.io/sig-storage/csi-snapshotter:v6.3.2
                                                                         registry.k8s.io/sig-storage/csi-resizer:v1.9.2
                                                                         registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.1
                                                                         dellemc/sdc:4.5
 
-dellemc/csi-vxflexos:v2.8.0     k8s1.27,k8s1.26,k8s1.25,ocp4.13,ocp4.12 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+quay.io/dell/container-storage-modules/csi-vxflexos:v2.12.0     k8s1.31,k8s1.30,k8s1.29,ocp4.17,ocp4.16 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
```

Comment:
Why are we repeating 2.12.0 again? @anandrajak1 aren't we only displaying latest going forward?

Reply:
Yes, lets go with latest version alone.